src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -282,19 +340,13 @@ protected void validateSchema(Table table, Schema tableSchema, boolean allowSche
       return;
     }
 
-    if (getConfig().isTruncateTableSet()) {
```

Comment:
Should we be worried about backward compatibility here? cc: @albertshau

Reply:
As per offline discussion, please call this out in the release notes.

Reply:
Will do.